xbmc/pvr/PVRManager.cpp

```diff
@@ -627,15 +627,16 @@ bool CPVRManager::Load(void)
   /* start the add-on update thread */
   if (m_addons)
     m_addons->Start();
+  else
+    return false;
 
-  /* load at least one client */
-  while (IsInitialising() && m_addons && !m_addons->HasConnectedClients())
-    Sleep(50);
```

Comment:
I specifically stated in the PR description that I know this is not the best way but that we need some kind of fix for the next release.